### ISSUE TITLE
Open Project Discussion Thread

### ISSUE THREAD

**Issue by droseblockchain** [CatholicBlockchainOrg/DonationTracking#1]
How could we enable people to donate to Catholic causes, organizations, and projects while giving transparency to where their donations are going, how they are being used?

Share your thoughts below on how we could build this project!

**Comment by mmastroi** [comment]
When I read the top 5 list, one idea was disaster response funds for Catholic charities. Ex: volcano recovery in central America. Someone with assets or cryto currency wants to help. We offer an app that enables these funds to purchase disaster resources quickly. 
Mark Mastroianni:
2-sided prod Dev challenge. 1: need mechanism to receive cryto donations. 2: need partners who will use cryto for water, medicine, generators, tents,...

**Comment by droseblockchain** [comment]
Good idea @mmastroi ! This repo could include such an app, disaster recovery. I also would suggest we look into leveraging/using/building on top of the partnership that MakerDAO + Dether + Bifrost are doing in this area: https://twitter.com/MakerDAO/status/1001515178414956545

We can add the Catholic specifics on it.

**Comment by slomcand** [comment]
I'm not sure if the idea for this topic. My proposes to extend the concept to active help: Support by working for Catholics who are poor but are talented and come from poor countries where they are persecuted and unable to find a job. Several tasks:
1. authorization of a person and his account on blockchain by a local Catholic community
2. confidential job order, including it can be language learning for a person from a different, richer country or other services that can be performed remotely and payable in cryptocurrencies

**Comment by slomcand** [comment]
Enabling transparent financing of blockchain projects and voting on the most needed solutions.
